external/fv3fit/fv3fit/keras/_models/shared/output_range.py

```diff
@@ -0,0 +1,71 @@
+import dataclasses
+import numpy as np
+import tensorflow as tf
+from typing import Optional, Union, Mapping, Hashable, Sequence
+
+
+Output = Union[np.ndarray, tf.Tensor]
+
+
+@dataclasses.dataclass
+class Range:
```

Comment:
Can you call this `OutputRange`, `Limits`, or something else to distinguish it from the `range` keyword?